beacon-chain/rpc/prysm/v1alpha1/validator/aggregator.go

```diff
@@ -145,14 +154,21 @@ func (vs *Server) SubmitAggregateSelectionProofElectra(
 		return nil, status.Errorf(codes.InvalidArgument, "Validator is not an aggregator")
 	}
 
-	atts := vs.AttPool.AggregatedAttestationsBySlotIndexElectra(ctx, req.Slot, req.CommitteeIndex)
-	if len(atts) == 0 {
-		atts = vs.AttPool.UnaggregatedAttestationsBySlotIndexElectra(ctx, req.Slot, req.CommitteeIndex)
+	var atts []*ethpb.AttestationElectra
+
+	if features.Get().EnableExperimentalAttestationPool {
+		atts = cache.GetBySlotAndCommitteeIndex[*ethpb.AttestationElectra](vs.AttestationCache, req.Slot, req.CommitteeIndex)
+	} else {
+		atts = vs.AttPool.AggregatedAttestationsBySlotIndexElectra(ctx, req.Slot, req.CommitteeIndex)
 		if len(atts) == 0 {
-			return nil, status.Errorf(codes.NotFound, "No attestations found in pool")
+			atts = vs.AttPool.UnaggregatedAttestationsBySlotIndexElectra(ctx, req.Slot, req.CommitteeIndex)
```

Comment:
Same as above: do we need to aggregate them?

Reply:
The behavior is the same, you must be reading something incorrectly